packages/jest-source-map/src/getCallsite.ts

```diff
@@ -66,3 +68,42 @@ export default (
 
   return stack;
 };
+
+export const getSourceMappedStack = (
+  level: number,
+  sourceMaps ?: SourceMapRegistry | null,
+): string => {
+
+  const rawStack = new ErrorWithStack(
+    undefined, 
+    getSourceMappedStack)
+    .stack || '';
+
+  const stackUtils = new StackUtils({
```

Comment:
do we need this? we have installed `source-map-support`, so any errors created should have its stack mapped back over sourcemaps already

Reply:
Thank you for pointing this out, `source-map-support` already takes care of what I tried to do here. I will remove this method altogether. 

Given the stacks created are source-mapped already, `BufferedConsole.write()` would not need `sourceMaps` as a parameter anymore. This seems a breaking change as well?

Reply:
I think it should be fine for the the consoles to just ignore the Buffered console - shouldn't be breaking